orchestrator-salt/src/main/resources/salt/salt/postgresql/upgrade/scripts/backup_db.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+# backup_db.sh
+# This script is needed as Azure does not support in place RDS upgrade
+# This script:
+#  - creates backup local temp directory
```

Comment:
A question: have you considered disk space issues, or are you going to treat that somewhere else separately?

Reply:
Yes, thank you, thought about it, and validations would be a separate issue as it seems to be a bigger chunk